Library/Homebrew/rubocops/lines_cop.rb

```diff
@@ -67,7 +67,71 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             next unless block_arg.source.size>1
             problem "\"inreplace <filenames> do |s|\" is preferred over \"|#{block_arg.source}|\"."
           end
+
+          [:rebuild, :version_scheme].each do |method_name|
+            find_method_with_args(body_node, method_name, 0) do
+              problem "'#{method_name} 0' should be removed"
+            end
+          end
+
+          [:mac?, :linux?].each do |method_name|
+            next unless formula_tap == "homebrew-core"
+            find_instance_method_call(body_node, "OS", method_name) do |check|
+              problem "Don't use #{check.source}; Homebrew/core only supports macOS"
+            end
+          end
+
+          find_method_with_args(body_node, :fails_with, :llvm) do
+            problem "'fails_with :llvm' is now a no-op so should be removed"
+          end
+
+          find_method_with_args(body_node, :system, /^(otool|install_name_tool|lipo)$/) do
+            next if @formula_name == "Cctools"
+            problem "Use ruby-macho instead of calling #{@offensive_node.source}"
+          end
+
+          find_every_method_call_by_name(body_node, :system).each do |method_node|
+            next if @formula_name =~ /^Kibana(\@\d+(\.\d+)?)?$/
+            first_param, second_param = parameters(method_node)
+            next if !node_equals?(first_param, "npm") ||
+                    !node_equals?(second_param, "install")
+            offending_node(method_node)
+            problem "Use Language::Node for npm install args" unless languageNodeModule?(method_node)
+          end
+
+          if find_method_def(body_node, :test)
+            problem "Use new-style test definitions (test do)"
+          end
+
+          if find_method_def(body_node, :options)
+            problem "Use new-style option definitions"
+          end
+
+          find_method_with_args(body_node, :skip_clean, :all) do
+            problem <<-EOS.undent.chomp
+              `skip_clean :all` is deprecated; brew no longer strips symbols
+                      Pass explicit paths to prevent Homebrew from removing empty folders.
+            EOS
+          end
+
+          find_instance_method_call(body_node, :build, :universal?) do
+            next if @formula_name == "Wine"
+            problem "macOS has been 64-bit only so build.universal? is deprecated."
```

Comment:
Should this be "...has been 64-bit only since 10.6..."?

Reply:
👍